src/platform/user/authentication/components/VerifyButton.jsx

```diff
@@ -55,50 +80,58 @@ export const VerifyIdmeButton = ({ queryParams }) => {
 };
 
 /**
- *
- * @returns The updated design of the Login.gov identity-verification buttion
+ * @returns Login.gov Verification Button
  */
 export const VerifyLogingovButton = ({ queryParams }) => {
   const { image, policy } = SERVICE_PROVIDERS.logingov;
-  const useOAuth = useSelector(isAuthenticatedWithOAuth);
+
+  const handleClick = async () => {
+    await verifyHandler({ policy, queryParams });
+  };
```

Comment:
Let's revert this back and have a fallback of `forceOAuth` for the `useOAuth` useSelector

`const useOAuth = useSelector(isAuthenticatedWithOAuth) || forceOAuth`